.github/workflows/ci.yml

```diff
@@ -58,6 +58,13 @@ jobs:
               env:
                   XDEBUG_MODE: coverage
 
+            - name: SonarQube Scan
```

Comment:
The replacement should occur in `sonar-cloud.yml` instead of adding the new step to `ci.yml`.